corpus/config/models.py

```diff
@@ -69,6 +69,24 @@ class Meta:
         verbose_name = "Society"
         verbose_name_plural = "Societies"
 
+class SIG(models.Model):
+    IEEE_SIGs=[
+        ("CompSoc","CompSoc"),
+        ("Diode","Diode"),
+        ("Piston","Piston"),
+        ("WiE","WiE"),
+        ("SIGHT","SIGHT"),
+    ]
+    name = models.CharField(verbose_name="Name", max_length=10, unique=True, choices=IEEE_SIGs)
```

Comment:
I don't think we need to keep choices. Let it be a normal field.

Reply:
done